tree/dataframe/test/TwoFloatsLinkDef.h

```diff
@@ -0,0 +1,6 @@
+#ifdef __CINT__
```

Comment:
Can be just replaced by `#ifdef __ROOTCLING__` :)

Reply:
done!